KoreBuild-dotnet/build/KoreBuild.cmd

```diff
@@ -14,7 +14,7 @@ IF "%KOREBUILD_DOTNET_CHANNEL%"=="" (
 )
 
 IF "%KOREBUILD_DOTNET_VERSION%"=="" (
-    SET KOREBUILD_DOTNET_VERSION=1.0.0.001496
+    SET /P KOREBUILD_DOTNET_VERSION=<win.cli.version
```

Comment:
A little picky but you should call the file `cli.version.<os>` to keep all files together when you sort by name